src/config/github-api.ts

```diff
@@ -1,12 +1,11 @@
 import { GitHubAPI } from "probot";
 import { getLogger } from "./logger";
-import { enhanceOctokit } from "./enhance-octokit";
 import { Options } from "probot/lib/github";
 
 export const GithubAPI = (options: Partial<Options> = {}): GitHubAPI => {
 	options.logger = options.logger || getLogger("config.github-api");
 	options.throttle = {
 		enabled: false
 	};
-	return enhanceOctokit(GitHubAPI(options as Options));
+	return GitHubAPI(options as Options);
```

Comment:
wait 😱 , will it be infinite recursion? Maybe try bellow for safety sake?
```javascript
import { GitHubAPI as ProbotGitHubAPI } from "probot";
...
	return ProbotGitHubAPI(options as Options);
```

Reply:
LOL, it wont' be infinite recursion, the `h` is `H` in probot! But still ...

Reply:
haha yeah its not ideal is it, only use on one class at the moment at least. will update it anyway with `ProbotGitHubAPI`